libs/s25main/gameData/PortraitConsts.h

```diff
@@ -15,18 +16,15 @@ struct PortraitInfo
     unsigned int resourceIndex;
 };
 
-constexpr std::array<PortraitInfo, 13> SUPPRESS_UNUSED Portraits = {
+constexpr auto SUPPRESS_UNUSED Portraits = helpers::make_array<PortraitInfo>(
```

Comment:
Just FYI: Without the template param it will use the common type of the arguments which is already `PotraitInfo`. Doesn't hurt though due to use of `auto` on the lhs